modules/installation-osp-kuryr-known-limitations.adoc

```diff
@@ -50,8 +43,6 @@ and UDP, are not supported.
 
 There are limitations when using Kuryr SDN that depend on your deployment environment.
 
-Because of Octavia's lack of support for the UDP protocol and multiple listeners, if the {rh-openstack} version is earlier than 13.0.13, Kuryr forces pods to use TCP for DNS resolution.
-
 In Go versions 1.12 and earlier, applications that are compiled with CGO support disabled use UDP only. In this case,
```

Comment:
This part until NOTE about musl should be removed too, it's tied to the DNS issue.

Reply:
Then note starting in line 87 should be gone too, 16 always supports UDP.